Gauge/Main/Options.hs

```diff
@@ -192,9 +194,16 @@ opts :: [OptDescr (Config -> Config)]
 opts =
     [ Option "I" ["ci"]         (ReqArg setCI "CI") "Confidence interval"
     , Option "G" ["no-gc"]      (NoArg setNoGC)     "Do not collect garbage between iterations"
-    , Option "L" ["time-limit"] (ReqArg setTimeLimit "SECS") "Time limit to run a benchmark, use 0 to force min-samples per benchmark"
-    , Option ""  ["min-samples"] (ReqArg setMinSamples "COUNT") "Minimum number of samples to be taken"
-    , Option ""  ["min-duration"] (ReqArg setMinDuration "MILLISECS") "Minimum duration of each sample, use 0 to force one iteration per sample"
+    , Option "L" ["time-limit"] (ReqArg setTimeLimit "SECS") $
+        "Min seconds for each benchmark run, default is "
```

Comment:
Should that be "Max seconds"?

Reply:
No. The confusion arises because of the name `time-limit`, I guess this option name comes from criterion. It is actual a minimum and not a maximum that's why I changed the wording in help.

see `runBenchmarkable'` for the use of `time-limit` and `minSamples`.

Reply:
It may have been `max` earlier, I think the meaning changed when I introduced `minSamples` option. Now with that option the test can run longer than `time-limit` to honor `minSamples`. So now, the max is actually the max of `minSamples` and `timeLimit`.

Reply:
Ideally we should change the name of this option (e.g. to `min-bench-duration`) to avoid confusion. But that will be a breaking change. Also the option `min-duration` may be confused with this and should perhaps be changed to `min-sample-duration`. Not sure if it will be ok to change these option names. @vincenthz ?

Reply:
maybe a good idea to keep the compat for the time being and adding a deprecation notice, along with the new one ?